annotationframeworkclient/auth.py

```diff
@@ -0,0 +1,178 @@
+import json
+import requests
+import webbrowser
+import os
+from .endpoints import auth_endpoints, default_server_address
+
+default_token_location = "~/.cloudvolume/secrets"
```

Comment:
maybe this should be replaced with os.environ.get(AUTHTOKEN_FOLDER, "~/.cloudvolume/secrets"), and default_token_name=os.environ.get(AUTHTOKEN_NAME, 'daf-secret.json') (as its not just a chunkedgraph token after all).

Reply:
I like that, will do